contents/blog/ab-testing-guide-for-engineers.md

```diff
@@ -0,0 +1,207 @@
+---
+date: 2023-07-04
+title: "A software engineer's guide to A/B testing"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ['lior-neu-ner']
+featuredImage: ../images/blog/experiment-hog.jpg
+featuredImageType: full
+featuredVideo: https://www.youtube-nocookie.com/embed/C2ICJbICfsg
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+---
+
+A/B testing enables you to be confident that your product changes are having their intended effects. Rather than making decisions based on gut feeling or intuition, A/B testing provides empirical data to help you remove bias from decision making.
+
+Companies like [Netflix](https://uxdesign.cc/how-netflix-does-a-b-testing-87df9f9bf57c), [Uber](https://www.uber.com/en-GB/blog/supercharging-a-b-testing-at-uber/) and [Booking.com](https://booking.ai/tagged/ab-testing) use A/B testing to ship hundreds of changes a day. In this blog, we'll teach you basic concepts to get you started in A/B testing and supercharge your product growth!
+
+> **Note:** A/B tests are commonly referred to as "experiments".
+
+## How A/B testing works
+
+In A/B testing, you first decide on a goal metric and what change you'd like to see. For example, you may to increase conversion rate, or you may want to decrease customer churn rate. Then, once you've made your code changes and are ready to test, you **randomly** split your users into two (or more) groups:
+
+- The control group: these users will experience your product as is – without any of your new changes.
+- The test group: these users will experience your product with your new changes.
+
+You then run your A/B test for some time, usually a few days or weeks, to gather data and compare the differences in the goal metric between the different groups.
```

Comment:
A little bit more on calculating how long could be valuable here.

Reply:
This is covered more below. But based on this (and some of your other feedback), I think it would be useful to outline what topics will be covered in the intro, so to give an idea of when to expect each topic